pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -132,10 +132,11 @@ def advance(self, batch, batch_idx):
             for k in range(len(batch_outputs)):
                 self.batch_outputs[k].extend(batch_outputs[k])
         else:
-            # in manual optimization, there is no looping over optimizers
-            result = self._run_optimization(batch_idx, split_batch)
-            if result:
-                self.batch_outputs[0].append(deepcopy(result.result_collection))
+            # in manual optimization, hand over execution to the ManualOptimization loop
+            output = self.manual_loop.run(split_batch, batch_idx, self._hiddens)
+            if output:
+                result, self._hiddens = output
```

Comment:
Who would use `manual_optimization` and tbptt?

I don't think we have a single test with manual optimization and tbptt enabled. Can you add one?

Reply:
I updated one test case to use manual optimization and automatic optimization.